ibodata/lateral_profile.py

```diff
@@ -120,7 +120,26 @@ def normalize(self, dt, allow_cast=True):
         mid = self.x_at_y(a) + w / 2.0
         self.x -= mid
 
-        ave = np.average(self.y[np.fabs(self.x) <= dt])
+        norm_section_y = self.y[np.fabs(self.x) <= dt]
+        norm_section_x = self.x[np.fabs(self.x) <= dt]
+        area = np.trapz(norm_section_y, norm_section_x)
+
+        """
+        if there's no point on the edge normalization is not perfectly accurate
+        and we are normalizing over smaller area than [-dt, +dt]
+        That's why we interpolate points on the edge below.
+        """
+        if np.argwhere(self.x == -dt).size == 0:
+            coords_y = (self.y_at_x(-dt), norm_section_y[0])
+            coords_x = (-dt, norm_section_x[0])
+            area += np.trapz(coords_y, coords_x)
```

Comment:
numpy `trapz` has a counterintuitive way of specifying parameters. I'd expect to call it like: `trapz(x,y)`, not `trapz(y,x)`.
To be more specific please use named arguments and change the order of arguments (which is possible when arguments are named):

```
area += np.trapz(x = coords_x, y = coords_y)
```

This will help future developers

Reply:
Done